ani-cli

```diff
@@ -628,6 +629,11 @@ if [ -z "$select_first" ]; then
 			ep_choice_start=$((episode + 1))
 			ep_choice_end=""
 			;;
+        	b)
```

Comment:
try using tabs here to maintain uniform indent

Reply:
I was using tabs before the new commit